plugins/api/src/switch_user.cpp

```diff
@@ -121,24 +121,36 @@ namespace
         }
     } // update_privilege_level
 
-    auto update_server_to_server_connections(RsComm& _comm) -> void
+    auto update_server_to_server_connections(RsComm& _comm, const SwitchUserInput& _input) -> void
     {
-        const auto switch_user_or_disconnect = [&client = _comm.clientUser](rodsServerHost& _host) {
+        const auto switch_user_or_disconnect = [&](rodsServerHost& _host) {
             if (!_host.conn) {
                 log_api::trace("No connection to remote host [{}].", _host.hostName->name);
                 return;
             }
 
-            // Disconnect if the remote server's version is older than 4.3.1.
+            // Disconnect if the client requested that all server-to-server connections be closed or
+            // if the remote server's version is older than 4.3.1.
```

Comment:
'older than' -> 'less than' ?

or 'prior to'

or 'earlier than'... as i see 'later' just below in the next couple lines...

Reply:
Good catch. This should be "earlier than".

Will change.

Reply:
done